component/authenticator/pom.xml

```diff
@@ -23,7 +23,7 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <artifactId>org.wso2.carbon.extension.identity.authenticator.duo.connector</artifactId>
-    <version>1.1.18-SNAPSHOT</version>
+    <version>1.1.17</version>
```

Comment:
What's the reason for changing this ?

Reply:
resolved with 76916f91a28ab923f57fa4ffa05093fd5c150b24